fe/src/main/java/org/apache/doris/qe/MultiLoadMgr.java

```diff
@@ -379,10 +378,14 @@ public DataDescription toDataDesc() throws DdlException {
                         throw new DdlException(e.getMessage());
                     }
                 }
+                format = properties.get(LoadStmt.KEY_IN_PARAM_FORMAT_TYPE);
+                if (Strings.isNullOrEmpty(format)) {
+                    format = "csv";
+                }
             }
 
             DataDescription dataDescription = new DataDescription(
-                    tbl, null, files, columns, columnSeparator, false, null);
+                    tbl, null, files, columns, columnSeparator, format, false, null);
```

Comment:
there is no need to change this file, because there is a DataDescription constructor without format